### ISSUE TITLE
404 error when navigate to /admin

### ISSUE THREAD

**Issue by danielbidala** [Laravel-Backpack/Base#318]
Hi Guys

After a fresh install (everything was fine) I get 404 error when navigate to http://mydomain/admin. In the config I din't modified 'route_prefix' => 'admin',

Any suggestion?

I'm on shared host.
Backpack 3.4
laravel 5.6
Php 7.1

**Comment by tabacitu** [comment]
Just double-checked, doesn't seem to do this on two different systems. I expect it's a server configuration issue on your end - maybe .htaccess problem (you need ```index.php``` in the URL) - I know some shared hosts disable .htaccess for some reason.

Closing this since I couldn't confirm it's a bug, sorry.
Cheers!

**Comment by danielbidala** [comment]
Hi @tabacitu, thanks for your reply.

If anybody is in the same situation:

I spent one day to figure it out. There is lot of articles on the net which tell you how to [set up laravel on shared hostings](https://github.com/petehouston/laravel-deploy-on-shared-hosting). I made all the settings that needed with no luck. Than it turned out the problem is my index.php which had CHMOD 0664 for some reason. I modified it to 0644 and everything is working fine now.
